dev/devicelab/bin/tasks/flutter_engine_group_performance.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/tasks/perf_tests.dart' show ListStatistics;
+import 'package:path/path.dart' as path;
+
+const String _bundleName = 'dev.flutter.multipleflutters';
+const String _activityName = 'MainActivity';
+const int _numberOfIterations = 10;
+
+List<String> _addComponent(List<String> components, String value) {
+  final List<String> result = <String>[];
+  result.addAll(components);
+  result.add(value);
+  return result;
+}
+
+List<String> _addComponents(List<String> components, List<String> values) {
+  final List<String> result = <String>[];
+  result.addAll(components);
+  result.addAll(values);
+  return result;
+}
+
+Future<void> _run(String command, List<String> args, String cwd) async {
+  print('$command: $args');
+  final Process process =
+      await Process.start(command, args, workingDirectory: cwd);
+  final Future<dynamic> stdoutStream = stdout.addStream(process.stdout);
+  final Future<dynamic> stderrStream = stderr.addStream(process.stderr);
+  final int exitCode = await process.exitCode;
+  if (exitCode != 0) {
+    throw Exception('command "$command $args" had exit code: $exitCode');
+  }
+  await stdoutStream;
+  await stderrStream;
+}
+
+Future<void> _withApkInstall(
+    String apkPath, String bundleName, Function(AndroidDevice) body) async {
+  final DeviceDiscovery devices = DeviceDiscovery();
+  final AndroidDevice device = await devices.workingDevice as AndroidDevice;
+  await device.unlock();
+  await device.adb(<String>['install', '-r', apkPath]);
+  try {
+    await body(device);
+  } finally {
+    await device.adb(<String>['uninstall', bundleName]);
+  }
+}
+
+Future<TaskResult> _doTest() async {
+  try {
+    final List<String> scriptPathComponents = path.split(Platform.script.path);
+    final List<String> multipleFluttersPath =
+        scriptPathComponents.sublist(0, scriptPathComponents.length - 4);
+    multipleFluttersPath.addAll(<String>['benchmarks', 'multiple_flutters']);
+    final String modulePath =
+        path.joinAll(_addComponent(multipleFluttersPath, 'module'));
+    final String androidPath =
+        path.joinAll(_addComponent(multipleFluttersPath, 'android'));
```

Comment:
Done, just so you know that doesn't really calculate the directory of the flutter folder, it just assumes it's in a certain location relative to where you are running the script.  The calculation I'm replacing is more robust since it is based on a known location of where the script is located.

Reply:
Hm, I wonder why it's off `Directory.current`, I thought it was off `Platform.script.toFilePath()`.